src/routes/safe/components/Apps/utils.js

```diff
@@ -29,40 +29,45 @@ export const getAppInfoFromUrl = async (appUrl: string) => {
     return res
   }
 
-  let cleanedUpAppUrl = appUrl.trim()
-  if (cleanedUpAppUrl.substr(-1) === '/') {
-    cleanedUpAppUrl = cleanedUpAppUrl.substr(0, cleanedUpAppUrl.length - 1)
-    res.url = cleanedUpAppUrl
+  res.url = appUrl.trim()
+  let noTrailingSlashUrl = res.url
```

Comment:
I like the usage of the URL class above, maybe we should use it here too, to build a valid url without the need to remove slashes. But for now I am also fine with the approach that I wrote here :)

Reply:
I didn't do it here, because it's not returning the complete URL if you call `new URL('manifest.json', res.url)`. If `res.url =  https://someUrl/compound/` the resulting URL is `https://someUrl/manifest.json/` in order to make it work, you have to do it like so: `new URL('/compound/manifest.json', 'https://someUrl')`